restapi.go

```diff
@@ -1390,7 +1390,7 @@ func (s *Session) GuildEmojiEdit(guildID, emojiID, name string, roles []string)
 
 	data := struct {
 		Name  string   `json:"name"`
-		Roles []string `json:"roles,omitempty"`
+		Roles []string `json:"roles"`
```

Comment:
While it would be a breaking change, I think that using `*[]string` (both for the `roles` parameter and field) would be better.